docs/components/dividers.md

```diff
@@ -0,0 +1,279 @@
+
+#### Inset divider example
+
+- [Class definition](https://api.flutter.dev/flutter/material/Divider-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/divider.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/dividers/inset_divider&gh_ref=develop)
+
+The following example shows two lists separated by an inset divider and a subheader
+
+![Inset divider](assets/dividers/inset_divider.png)
+
+```dart
+Divider(indent: 16),
+Container(
+  padding: EdgeInsets.only(left: 16),
+  child: Align(
+    alignment: Alignment.centerLeft,
+    child: Text(
+      'Subheader',
+      style: Theme.of(context).textTheme.bodyText2.copyWith(
+          fontSize: 12.0,
+          color: Theme.of(context).textTheme.caption.color),
+      textAlign: TextAlign.start,
+    ),
+  ),
+),
+```
+
+### Middle dividers
+
+Middle dividers space related content and are centered in a layout or list.
+
+#### Middle divider example
+
+- [Class definition](https://api.flutter.dev/flutter/material/Divider-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/divider.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/dividers/middle_divider&gh_ref=develop)
+
+The following example shows a middle divider separating a list of items and their prices and the total cost:
+
+![Middle divider](assets/dividers/middle_divider.png)
+
+```dart
+Divider(
+  indent: 16,
+  endIndent: 16,
+),
+```
+
+### Flutter Vertical Divider
+
+Vertical divider offers the same parameters as the regular divider but instead the divider is vertical.
+
+`VerticalDivider`
+
+- [Class definition](https://api.flutter.dev/flutter/material/VerticalDivider-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/divider.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/dividers/vertical_divider&gh_ref=develop)
+
+![Vertical divider](assets/dividers/vertical_divider.png)
+
+```dart
+VerticalDivider()
+```
+
+## Theming dividers
+
+`Divider`
+
+- [Class definition](https://api.flutter.dev/flutter/material/Divider-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/divider.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/dividers/theme&gh_ref=develop)
+
+The following shows an inset divider with [Shrine theming](https://material.io/design/material-studies/shrine.html):0
```

Comment:
There's an extraneous "0" at the end of the phrase.

Reply:
Done